app/src/split/bluetooth/peripheral.c

```diff
@@ -43,15 +43,57 @@ static const struct bt_data zmk_ble_ad[] = {
 
 static bool is_connected = false;
 
-static int start_advertising() {
-    return bt_le_adv_start(BT_LE_ADV_CONN, zmk_ble_ad, ARRAY_SIZE(zmk_ble_ad), NULL, 0);
+static void each_bond(const struct bt_bond_info *info, void *user_data) {
+    bt_addr_le_t *addr = (bt_addr_le_t *)user_data;
+
+    char addr_str[BT_ADDR_LE_STR_LEN];
+
+    bt_addr_le_to_str(&info->addr, addr_str, sizeof(addr_str));
```

Comment:
This appears to be unused now that the logging is removed.

Reply:
Indeed. Removed.